adrs/README.md

```diff
@@ -0,0 +1,21 @@
+# Architecture Decision Records (ADRs)
+An architecture decision record (ADR) is a document that captures an important architecture decision made along with its context and consequences. We strive to keep these records small and lean.
+
+### Goal
+We want to record important design decisions to help current and future contributors of the library to understand the reasons for doing things a certain way.
+
+
+### Lightweight Process
+
+- Create a new branch on this repository and copy `adrs/0000-template.md` to a new file with the next free number.
+- Modify the template to add details about your proposal and add any supporting picture under the `adrs/images` folder.
+- Make a Pull Request (PR) for your branch.
+- Once consensus is reached and approvals given using the Github review system, the PR can be merged.
+
+In case the team has already agreed on a decision and no further async discussion is necessary just go ahead an record it directly without opening any PR.
+
+
+### Disclaimer
+
+As of today, the primary audience for these ADRs artifacts is the Stacks Core Team. If circumstances will change and we feel a broader audience should get involved in the decision making process we could look into evolving our process in something a bit more formal like the internal [Stack Overflow RFC process](https://github.com/StackEng/StackOverflow/blob/master/docs/rfcs/0001-lightweight-rfc-process.md).
```

Comment:
nit: We use the term "Core" (capital C) to refer to the Stack Overflow "Core" codebase. I've further tweaked the wording here to match my personal tastes. Let me know if you disagree and we can change things back.

```suggestion
As of today, the primary audience for these ADRs artifacts is the core Stacks Team. If circumstances change and we feel a broader audience should get involved in the decision making process we could look into evolving our process in something a bit more formal like the internal Stack Overflow RFC process.
```

Side note: I removed the link to the private/internal SO repo. I could go either way on this, but something about linking to a private repo in an OSS project doesn't sit right with me.

Reply:
Thanks for clarifying. Happy to remove the direct link.